src/app/components/challenge/challengeoverview/challengeoverview.component.ts

```diff
@@ -32,5 +42,39 @@ export class ChallengeoverviewComponent implements OnInit {
     challenge => {
       this.challenge = challenge;
     });
+    this.challengeService.isChallengeHost.subscribe(status => {
+      this.isChallengeHost = status;
+    });
+  }
+
+  editChallengeOverview() {
+    const SELF = this;
+
+    const apiCall = (params) => {
+      const BODY = JSON.stringify(params);
+      SELF.apiService.postUrl(
+        SELF.endpointsService.editChallengeDetailsURL(SELF.challenge.creator.id, SELF.challenge.id),
+        BODY
+      ).subscribe(
+          data => {
+            SELF.globalService.showToast('success', 'The challenge title is  successfully updated!', 5);
+
+          },
+          err => {
+            SELF.globalService.handleApiError(err, true);
+            SELF.globalService.showToast('error', err);
+          },
+          () => console.log('EDIT-CHALLENGE-DESCRIPTION-FINISHED')
+        );
+    };
+
+    const PARAMS = {
+      title: 'Edit Challenge Description',
+      content: 'asdasdasd \n\nasdasdasdasd',
```

Comment:
What's this for? @Sanji515

Reply:
@lunayach actually this will be inside the modal for editing the challenge description and have updated in the PR (#144)

Reply:
Okay. :+1: